### PR TITLE
Test and fix loqusdbapi sv call 

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

**How to test**:
1. check loqus sv freq on a variant - note it generally missing, even if the case in question is loaded
2. apply fix and see the variant reappear if loaded

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

Before:
Screenshot 2021-11-22 at 18 10 00

After:
Screenshot 2021-11-22 at 18 09 46

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
Ahem, but on my local the total counts also disappeared. Maybe some naming issue with the returned total counts on svs as well? Checking.

**Comment by dnil** [comment]
> Ahem, but on my local the total counts also disappeared. Maybe some naming issue with the returned total counts on svs as well? Checking.

Ah, ok, that was just on local. I have an old Docker somewhere, pre the first v1.0 where "total" is still "nr_cases". Works fine on stage:
Screenshot 2021-11-22 at 18 38 24

File: scout/server/extensions/loqus_extension.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def get_api_loqus_variant(api_url, variant_info):
             end = variant_info["end"]
 
             sv_type = variant_info["variant_type"]
-            search_url = f"{search_url}/svs/?chrom={chrom}&end_chrom={end_chrom}&pos={pos}&end={end}&sv_type={sv_type}"
+            search_url = f"{search_url}/?chrom={chrom}&end_chrom={end_chrom}&pos={pos}&end={end}&sv_type={sv_type}"
```
Comment (northwestwitch) [inline scout/server/extensions/loqus_extension.py:238]: This is interesting. Obviously an error I've introduced when I've created the support for the loqus API. But how come that this works if the loqusdbapi endpoint is shaped like this: https://github.com/Clinical-Genomics/loqusdbapi/blob/3fb1830e3b8f3d04a6e638f12d454e95475e8030/loqusdbapi/main.py#L98 ??

🤔

**Review by northwestwitch — COMMENTED** [review]
Didn't we test how to retrieve SVs in https://github.com/Clinical-Genomics/scout/pull/2349?

Look at the comments with the validations.. 🤔

File: scout/server/extensions/loqus_extension.py (hunk 1 above)
Comment (dnil) [reply scout/server/extensions/loqus_extension.py:238]: The search url already has the category here:
https://github.com/Clinical-Genomics/scout/blob/3995a367b4a5e56dd23e9ee172992cc74b69e6e0/scout/server/extensions/loqus_extension.py#L227

Comment (dnil) [reply scout/server/extensions/loqus_extension.py:238]: So it was added twice, basically, once explicitly once implicitly..

Comment (northwestwitch) [reply scout/server/extensions/loqus_extension.py:238]: Aha! I see now!

Comment (dnil) [reply scout/server/extensions/loqus_extension.py:238]: > Didn't we test how to retrieve SVs in #2349?
> 
> Look at the comments with the validations.. 🤔

Yes, and I also tested it in https://github.com/Clinical-Genomics/loqusdbapi/issues/19, where it failed first, then seemed to work. I think I am missing something about the connection session longevity: I guess both extensions must exist at startup, but perhaps we choose a little too early on a page which one to use?

Comment (northwestwitch) [inline scout/server/extensions/loqus_extension.py:238]: 🚀

**Review by northwestwitch — APPROVED** [review]
Of course, I see the error now. Good job! 💯

**Pull request merged**